client/src/components/PizzaPlaceShow.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react"
+
+const PizzaPlaceShow = props => {
+    const [pizzaPlace, setPizzaPlace] = useState({
+        name: "",
+        address: "",
+        phoneNumber: "",
+        website: "",
+        hours: "",
+        imageUrl: ""
+    })
+
+    const getPizzaPlace = async () => {
+        const pizzaId = props.match.params.id
+        try {
+            const response = await fetch(`/api/v1/pizza-places/${pizzaId}`)
+            if (!response.ok) {
+                throw new Error(`${response.status} (${response.statusText})`)
+            }
+            const pizzaData = await response.json()
+            setPizzaPlace(pizzaData.pizzaPlace)
+        } catch(error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getPizzaPlace()
+    }, [])
+
+    return (
+        <div className="show-page">
+            <h1>{pizzaPlace.name}</h1>
+            <div className="show-page-flex">
+                <div className="show-page-info">
+                    <p>Address: {pizzaPlace.address}</p>
+                    <p>Phone Number: {pizzaPlace.phoneNumber}</p>
+                    <p>Website: 
+                        <a href={pizzaPlace.website}> {pizzaPlace.website}</a>
```

Comment:
adding `target="_blank"` to an anchor tag will ensure that the link opens in a new browser tab instead of replacing the current page